### ISSUE TITLE
audio playback loading error

### ISSUE THREAD

**Issue by copkiller** [jsierles/react-native-audio#59]
I ran the AudioExample project. After pressing RECORD, STOP and PLAY, I get 

2016-04-01 03:14:24.691 AudioExample[5087:2446864] audio playback loading error: The operation couldn’t be completed. (OSStatus error 2003334207.)

I'm using

react-native -v
react-native-cli: 0.1.10
react-native: 0.21.0

**Comment by mouneyrac** [comment]
Hi I have this exact same error (#70) when I try to play a sound for which I am pretty sure I must be giving the wrong path. However I tried the audioexample (in May) and it worked perfectly.

**Comment by jsierles** [comment]
I'll take a look at this today.

**Comment by mouneyrac** [comment]
PS: I don't know if I was confusing, but when I said the audioexample worked in May, I didn't mean it stopped working (actually I believe it is still working as I am using the same code in an app). I just wanted to mention I had the same "audio playback loading error: The operation couldn’t be completed" error that copkiller reported. And I discovered it was due to the fact that I can not access files into React Native "js" folders with most thirdparty plugin (actually only the react-native require('image') seems to let you open a file into the "js" folders).
